sdk/ml/azure-ai-ml/azure/ai/ml/constants/_workspace.py

```diff
@@ -45,3 +45,9 @@ class OutboundRuleType:
     FQDN = "FQDN"
     PRIVATE_ENDPOINT = "PrivateEndpoint"
     SERVICE_TAG = "ServiceTag"
+
+
```

Comment:
mark this as experimental as well so that we're not stuck with it forever

Reply:
done